clustersConfig.py

```diff
@@ -185,7 +185,7 @@ def __init__(self, yaml_path: str, worker_range: common.RangeList):
         # creates hosts entries for each referenced node name
         node_names = {x["name"] for x in cc["hosts"]}
         for node in self.all_nodes():
-            if node.kind != "physical" and node.node not in node_names:
+            if node.node not in node_names:
```

Comment:
Why do we revert this?

Reply:
The idea is to automatically add a host config object for all nodes that don't have an explicitly configured one.  The `hosts` config was used only for machines that host VMs.  But it's easier if we have a host record per physical machine.  The code in the following commits should ensure that we only use this for VMs.

I do have a bug that I need to resolve though - trying to connect to all hosts directly, regardless if they're configured to run vms or not and configuring their virbr.

I hope it will be more clear when I push the next set of changes that fix this.